code/modules/cargo/bounties/engineering.dm

```diff
@@ -0,0 +1,50 @@
+/datum/bounty/item/engineering/pluoxium_tank
+	name = "Full Tank of Pluoxium"
+	description = "CentCom RnD is researching extra compact internals. Ship us a tank of Pluoxium and you'll be compensated."
+	reward = 7500 // Easy to produce as a supermatter engine byproduct
+	wanted_types = list(/obj/item/tank)
+	var/moles_required = 20 // A full tank is 28 moles, but CentCom ignores that fact.
+
+/datum/bounty/item/engineering/pluoxium_tank/applies_to(obj/O)
+	if(!..())
+		return FALSE
+	var/obj/item/tank/T = O
+	if(!T.air_contents.gases[/datum/gas/pluoxium])
+		return FALSE
+	return T.air_contents.gases[/datum/gas/pluoxium][MOLES] >= moles_required
+
+/datum/bounty/item/engineering/nitryl_tank
+	name = "Full Tank of Nitryl"
+	description = "The non-human staff of Station 88 has been volunteered to test performance enhancing drugs. Ship them a tank of Nitryl so they can get started."
+	reward = 15000 // Requires actual effort to produce
+	wanted_types = list(/obj/item/tank)
+	var/moles_required = 20
+
+/datum/bounty/item/engineering/nitryl_tank/applies_to(obj/O)
+	if(!..())
+		return FALSE
+	var/obj/item/tank/T = O
+	if(!T.air_contents.gases[/datum/gas/nitryl])
+		return FALSE
+	return T.air_contents.gases[/datum/gas/nitryl][MOLES] >= moles_required
+
+/datum/bounty/item/engineering/tritium_tank
+	name = "Full Tank of Tritium"
+	description = "Station 49 is looking to kickstart their research program. Ship them a tank full of Tritium."
+	reward = 20000 // Coveted by RnD, very useful for tritium bombs and thus hard to get hold of
```

Comment:
this is way too much for tritium. research tritium bombs use 50 mol at most, and using the incinerator you can get upwards of 10000. it isn't hard to get ahold of at all, and certainly shouldn't be more than nitryl or pluoxium.

also, consider making all these tank bounties a subtype like /datum/bounty/item/engineering/gas_tank
that way you can just set one variable to a different gas like tritium or pluoxium for each subtype, and keep the applies_to functionality in the parent type instead of copy pasted for every gas type.

Reply:
Thanks for your input, I haven't done atmos in ages and had no clue what the gases are actually worth.